etl/steps/data/garden/health/2025-07-16/drug_approvals.meta.yml

```diff
@@ -0,0 +1,49 @@
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Medicine & Biotechnology
+    processing_level: major
+
+
+dataset:
+  update_period_days: 365
+
+
+tables:
+  total_drug_approvals:
+    variables:
+      total_approvals:
+        title: New drugs approved by the FDA
+        unit: 'approvals'
+        short_unit: ''
+        description_short: |-
+          The annual number of new drugs approved by the FDA. This is broken down into [new chemical drugs](#dod:new-molecular-entity), new vaccines, and other new biologics such as antibodies and gene therapies. The data does not include [generics](#dod:generic-drug), reformulations, or [biosimilars](#dod:biosimilar).
+        display:
+          numDecimalPlaces: 0
+        description_processing: |-
+          - This data includes both [new molecular entities](#dod:new-molecular-entity) (NMEs) and [biologics license applications](#dod:biologics-license-application) (BLAs) approved by the Center for Drug Evaluation and Research (CDER) and the Center for Biologics Evaluation and Research (CBER) of the FDA.
+          - Biologicals approved by the CBER have been extracted from the FDA's "Purple Book", which lists licensed biological products with reference product exclusivity and biosimilarity or interchangeability evaluations.
+          - We removed biosimilars from the dataset by excluding products with a "biosimilar" or "interchangeable" license type as well as products with a reference product listed.
+        presentation:
+          grapher_config:
+            note: |-
+              This data includes “[BLAs](#dod:biologics-license-application)” (used for biologics and vaccines) and “[NMEs](#dod:new-molecular-entity)” (used for chemical drugs). Data on approvals of chemical drugs are only available from 1984.
```

Comment:
I'd have biologics license applications (BLAs) and new molecular entities (NMEs), spelled out. It's good to do this in case people don't notice the DoD or take a screenshot.